connect-inject/container_init.go

```diff
@@ -124,6 +133,9 @@ services {
     upstreams {
       destination_name = "{{ .Name }}"
       local_bind_port = {{ .LocalPort }}
+			{{ if .Datacenter }}
+			datacenter = "{{ .Datacenter }}"
```

Comment:
```suggestion
      datacenter = "{{ .Datacenter }}"
```
Aligns the indentations.